patches/webpack+4.46.0.patch

```diff
@@ -0,0 +1,144 @@
+diff --git a/node_modules/webpack/lib/Compilation.js b/node_modules/webpack/lib/Compilation.js
+index 6329174..196c4bb 100644
+--- a/node_modules/webpack/lib/Compilation.js
++++ b/node_modules/webpack/lib/Compilation.js
+@@ -1915,7 +1915,7 @@ class Compilation extends Tapable {
+ 		const hashFunction = outputOptions.hashFunction;
+ 		const hashDigest = outputOptions.hashDigest;
+ 		const hashDigestLength = outputOptions.hashDigestLength;
+-		const hash = createHash(hashFunction);
++		const hash = createHash('sha512');
```

Comment:
Is there a way to set the `hashFunction` in the webpack config instead of overriding the source code?

Reply:
https://github.com/adobe/react-spectrum/pull/4945#issuecomment-1692721796
> Using the patch-package I've come up with here, does work. I derived this from the later comment in this StackOverflow https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported where it's titled "Danger". I tried just passing the new hashing algorithm via the webpack config, however, it didn't appear to do anything, and there are several places where the wrong hash algorithm is hardcoded.